nbs/01_hello_world.ipynb

```diff
@@ -6,75 +6,14 @@
    "metadata": {},
```

Comment:
_Commented on notebook `nbs/01_hello_world.ipynb` Cell 31 Line 4_
```python
test.test_eq(
    None,
    cli(name="John", alternative_name="Jane", config_file="./config.default.env"),
)
```
I'd personally leave a test in. This is automatically run then for all inputs and is a way to demonstrate how to develop tests. It's not written on much as I wasn't pushing the idea before.